homeassistant/components/isy994/__init__.py

```diff
@@ -114,7 +114,7 @@
         "insteon_type": ["1."],
     },
     "switch": {
-        "uom": ["2", "78"],
+        "uom": ["2", "78", "100"],
```

Comment:
I'm a little hesitant about this one. 100 is a pretty generic uom in the ISY documentation.

I'm wondering if we can just go with the `insteon_type` and leave uom unchanged, as only one of these needs to match the node for the code to assign it to a domain.

Are you testing on 5.x or 4.x?

Reply:
Testing on `5.x`. `5.0.15` to be specific. Omitting `100` works just fine.

```suggestion
        "uom": ["2", "78"],
```